### PR TITLE
chore: Remove PSP support

### PR DESCRIPTION
**Pull request by almaslennikov**
Pod Security Policy was deprecated for a long time, and in k8s 1.25 it was completely dropped.
To align with the newest versions, dropping the PSP from our code

### CODE REVIEW

**Comment by abdallahyas** [comment]
retest-all

File: bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -608,15 +584,4 @@ spec:
   provider:
     name: NVIDIA
     url: https://github.com/Mellanox/network-operator/
-  relatedImages:
```
Comment (e0ne) [inline bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml:611]: related images shouldn't be deleted

**Review by adrianchiris — DISMISSED** [review]

File: bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (rollandf) [reply bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml:611]: Seems that the operator-sdk is now ignoring the relatedImages from the `config/manifests/bases`
It used to work in previous version...

Comment (rollandf) [reply bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml:611]: it stopped working in operator-sdk v1.19.1

**Comment by rollandf** [comment]
@almaslennikov can you rebase this one?

**Comment by almaslennikov** [comment]
> @almaslennikov can you rebase this one?

Done

**Comment by rollandf** [comment]
> > @almaslennikov can you rebase this one?
> 
> Done

Thanks! Can we remove the "hold" label?

**Comment by adrianchiris** [comment]
Merging per 2 LGTMs

**Review by e0ne — APPROVED** [review]

**Pull request merged**